### ISSUE TITLE
Drop-in 3.6.4, StringIndexOutOfBoundsException in DropInActivity

### ISSUE THREAD

**Issue by geertjan-tiqets** [Adyen/adyen-android#169]
Hi, I'm seeing a new crash when updating from 3.6.2 to 3.6.4:

```
02-24 10:30:10.931: E/MonitoringInstr(7840): Exception encountered by: com.adyen.checkout.dropin.ui.DropInActivity@80e22e6. Dumping thread state to outputs and pining for the fjords.
02-24 10:30:10.931: E/MonitoringInstr(7840): java.lang.StringIndexOutOfBoundsException: length=2; index=3
02-24 10:30:10.931: E/MonitoringInstr(7840): 	at java.lang.String.indexAndLength(String.java:294)
02-24 10:30:10.931: E/MonitoringInstr(7840): 	at java.lang.String.substring(String.java:1067)
02-24 10:30:10.931: E/MonitoringInstr(7840): 	at com.adyen.checkout.dropin.ui.DropInActivity.createLocalizedContext(DropInActivity.kt:172)
02-24 10:30:10.931: E/MonitoringInstr(7840): 	at com.adyen.checkout.dropin.ui.DropInActivity.attachBaseContext(DropInActivity.kt:127)
02-24 10:30:10.931: E/MonitoringInstr(7840): 	at android.app.Activity.attach(Activity.java:6188)
...
```

We're getting this crash when running our tests on Firebase Testlab. Apparently the Firebase device is using a Locale without a country.

I can reproduce the bug by forcing the locale by calling `DropInConfiguration.Builder.setShopperLocale(Locale.ENGLISH)`.

**Comment by xiaoxiaohua333** [comment]
3.6.4
--------
Caused by java.lang.StringIndexOutOfBoundsException: length=2; index=3
       at java.lang.String.substring + 1999(String.java:1999)
       at com.adyen.checkout.dropin.ui.DropInActivity.createLocalizedContext + 172(DropInActivity.java:172)
       at com.adyen.checkout.dropin.ui.DropInActivity.attachBaseContext + 127(DropInActivity.java:127)
       at android.app.Activity.attach + 7241(Activity.java:7241)
       at android.app.ActivityThread.performLaunchActivity + 3033(ActivityThread.java:3033)
       at android.app.ActivityThread.handleLaunchActivity + 3235(ActivityThread.java:3235)
       at android.app.servertransaction.LaunchActivityItem.execute + 78(LaunchActivityItem.java:78)
       at android.app.servertransaction.TransactionExecutor.executeCallbacks + 108(TransactionExecutor.java:108)
       at android.app.servertransaction.TransactionExecutor.execute + 68(TransactionExecutor.java:68)
       at android.app.ActivityThread$H.handleMessage + 1926(ActivityThread.java:1926)
       at android.os.Handler.dispatchMessage + 106(Handler.java:106)
       at android.os.Looper.loop + 214(Looper.java:214)
       at android.app.ActivityThread.main + 6986(ActivityThread.java:6986)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run + 494(RuntimeInit.java:494)
       at com.android.internal.os.ZygoteInit.main + 1445(ZygoteInit.java:1445)

**Comment by caiofaustino** [comment]
Thanks, we will fix this issue on the next release.

**Comment by caiofaustino** [comment]
Hi there,
Sorry for the delay but the fix is in the latest `3.6.5` release.

Let us know if you still have any problems.
